tests/Tests/HbBlobTest.cs

```diff
@@ -11,18 +11,18 @@ public class HbBlobTest : SKTest
 		[SkippableFact]
 		public void ShouldCreateFromFileName()
 		{
-			using (var blob = Blob.FromFile(Path.Combine(PathToFonts, "content-font.ttf")))
+			using (var blob = Blob.FromFile(Path.Combine(PathToFonts, "Distortable.ttf")))
 			{
-				Assert.Equal(246224, blob.Length);
+				Assert.Equal(16384, blob.Length);
 			}
 		}
 
 		[SkippableFact]
 		public void ShouldCreateFromStream()
 		{
-			using (var blob = Blob.FromStream(File.Open(Path.Combine(PathToFonts, "content-font.ttf"), FileMode.Open, FileAccess.Read)))
+			using (var blob = Blob.FromStream(File.Open(Path.Combine(PathToFonts, "Funkster.ttf"), FileMode.Open, FileAccess.Read)))
 			{
-				Assert.Equal(246224, blob.Length);
+				Assert.Equal(236808, blob.Length);
```

Comment:
Rather add new tests, or use the `[SkippableTheory]` with `[InlineData]` to keep the old tests. I think the theory way might be better for future tests.

What was the reason for this particular change?

Reply:
Parallel access to a font file was causing issues so I use a different file for that tests. Didn't know how to avoid that. We could add font files as resource to avoid this I guess.

Reply:
Ah. Yeah. I think there was an issue in skia somewhere that I noticed that it sometimes didn't close the file streams properly.